py3status/core.py

```diff
@@ -641,6 +634,30 @@ def setup(self):
             sys.stdout = Path("/dev/null").open("w")
             sys.stderr = Path("/dev/null").open("w")
 
+        # make sure we honor custom i3bar protocol stop/resume signals
+        # while providing users a way to opt out from that feature
+        # using the 0 value as specified by the i3bar protocol
+        custom_stop_signal = (
+            self.config["py3_config"].get("py3status", {}).get("stop_signal")
+        )
+        if custom_stop_signal is not None:
+            if isinstance(custom_stop_signal, int):
+                # 0 is a special value for i3bar protocol, use it as-is
+                if custom_stop_signal == 0:
+                    self.stop_signal = custom_stop_signal
+                else:
+                    self.stop_signal = Signals(custom_stop_signal)
+            else:
+                self.stop_signal = Signals[custom_stop_signal]
```

Comment:
(Imo) Maybe we shouldn't support strings... kinda like how we supported button strings, then later switched to button number because there can be  more input button numbers without strings.

If there is something from user, try it as-is, the input should fail on `0`, above `31`, and on any other inputs. 

By avoiding stings and any other non-int inputs, the lines would be simplified to something like this. Idk.
```python
if custom_stop_signal is not None:
    try:
        self.stop_signal = Signals(custom_stop_signal)
    except ValueError:
        self.stop_signal = 0

Reply:
k, I added a meaningful error as well